client/packages/common/src/intl/locales/tet/common.json

```diff
@@ -345,7 +345,7 @@
   "label.requested": "Kuantidade husu",
   "label.requisition": "Requizisaun",
   "label.rnr-adjustments": "Ajustamentu",
-  "label.rnr-consumed": "Kuantitade konsumu / distribuída",
+  "label.rnr-consumed": "Distribution",
```

Comment:
Surely this should be `Distribuída` or something?

Reply:
: )
![Screenshot 2025-05-16 at 5 11 21 PM](https://github.com/user-attachments/assets/636f3caa-aee5-4a5e-882f-6c6ef70b7e08)